contracts/eosio.system/src/producer_pay.cpp

```diff
@@ -43,7 +43,7 @@ namespace eosiosystem {
 
       /// only update block producers once every minute, block_timestamp is in half seconds
       if( timestamp.slot - _gstate.last_producer_schedule_update.slot > 120 ) {
-         update_elected_producers( timestamp );
+         update_elected_producers( timestamp, bheader.previous );
```

Comment:
@porkchop I don't know if this is correct (use _block_header::previous_ as the entry hash in our "random" system)

Reply:
Right now I think this block hash (the pointer to the previous block) is as good as any. Let's go with this at least for the time being